### PR TITLE
Removed invitataions from old GET endpoints.

### PR DESCRIPTION
**Pull request by krishna-patel-ch**
__Short description outlining key changes/additions__

__JIRA Ticket Number__

IDVA6-1064

### Type of change

* [ ] Bug fix
* [x] New feature
* [ ] Breaking change
* [x] Infrastructure change

### Pull request checklist

* [x] I have added unit tests for new code that I have added
* [x] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/main/java/uk/gov/companieshouse/accounts/association/utils/MapperUtil.java [hunk 1]
```diff
@@ -54,22 +54,6 @@ public Association enrichAssociationWithUserDetails(final Association associatio
         return association;
     }
 
-    public Association enrichInvitations(final Association association) {
```
Comment (kkonuganti-ch) [inline src/main/java/uk/gov/companieshouse/accounts/association/utils/MapperUtil.java:57]: you would still need this for invitations endpoint.

Comment (krishna-patel-ch) [reply src/main/java/uk/gov/companieshouse/accounts/association/utils/MapperUtil.java:57]: Good point - added it back

**Review by kkonuganti-ch — APPROVED** [review]

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Pull request merged**